src/Panel/panel.scss

```diff
@@ -35,42 +35,43 @@
 
   .panel-soft-loader {
     position: absolute;
-    width: 100%;
+    z-index: 999;
     top: 0;
+    width: 100%;
     height: 1px;
-    z-index: 999;
 
     .loader {
+      position: relative;
       width: 100%;
       height: 1px;
       background-color: $aquamarine30;
-      position: relative;
 
       &:after {
-        content: '';
         position: absolute;
+        right: 100%;
+        left: 0;
         display: block;
         height: 100%;
-        background-color: $aquamarine;
-        left: 0%;
-        right: 100%;
+        content: '';
         animation: loading 2s ease infinite;
+        background-color: $aquamarine;
       }
     }
   }
 
   > .panel-header {
-    background-color: $charcoal;
+    overflow: hidden;
     color: $white;
-    padding-left: 20px;
+    background-color: $charcoal;
 
     .expand-icon {
       font-size: 20px;
     }
 
     .panel-header-title {
-      font-weight: $semibold;
       font-size: 16px;
+      font-weight: $semibold;
+      padding-left: 20px;
```

Comment:
Move from `.panel-header` to `.panel-header-title` cause as long as `.panel` hasn't a `min-width` specified when it is shrank an area of 20px (`padding-left: 20px`) remains visible.